cmd_init/cmd_init.go

```diff
@@ -0,0 +1,13 @@
+/*
+The purpose of this file is to initialize all of the commands, that are spread across different folders,
+and to keep the imports in one place, keeping main.go clean.
+*/
+
+package cmd_init
+
+import (
+	_ "github.com/exoscale/cli/cmd/compute/load_balancer"
+)
+
+func InitCommands() {
```

Comment:
...maybe someone has a better suggestion though...but seems like we would need to import _every_ new package somewhere

Reply:
i was planning to put it under main actually, but then i get `# command-line-arguments
./main.go:34:2: undefined: InitCommands****`
no idea why

Reply:
ah, nvm..i think i figured out what you mean